Dockerfile

```diff
@@ -29,13 +29,18 @@ RUN wget -qO redis.tar.gz https://github.com/antirez/redis/archive/${redis_versi
     && tar xfz redis.tar.gz -C / \
     && mv /redis-$redis_version /redis
 
-RUN (cd /redis && make)
+RUN (cd /redis && export BUILD_TLS=yes && make)
```

Comment:
This will trigger a rebuild of all images. More a general question, do this flag works for all redis versions back to 3.0.0 in the same way?

You MUST at least verify that this flag works for at least one version in each major redis-server version, so 3.0, 3.2, 4.0, 5.0, 6.0 please.